vision.py

```diff
@@ -104,13 +104,11 @@ def process(frame: np.ndarray, mask: np.ndarray = None, hsv: np.ndarray = None,
 
             # find the centre of the contour
             M = cv2.moments(contour)
-            x, y = M["m10"] / M["m00"], M["m01"] / M["m00"]
+            x, y = M["m10"] / M["m00"]
```

Comment:
Are you sure this works? Because I'm not.

Reply:
Oops, yeah I'll fix that.